Decred Wallet/Extensions/UITableView.swift

```diff
@@ -15,13 +16,21 @@ extension UITableView {
             return false
         }
         
-        // Only return true if cell is fully displayed, i.e. cell top + cell height <= table height.
+        // Only return true if cell is@objc  fully displayed, i.e. cell top + cell height <= table height.
         let cellScrollPos = cell.frame.origin.y - self.contentOffset.y
         return cellScrollPos + cell.frame.height <= self.frame.height
     }
 }
 
 extension UITableViewCell {
+    class var identifier: String {
+        return String.className(self)
+    }
+    
+    @objc open class func height() -> CGFloat {
+        return 48
```

Comment:
this extension and functions are from the old `slidemenucontroller`. It ended up being used in the Transaction history and every other table. Deprecating the slidemenu controller caused a whole bunch of errors relating to the `height` function being called on those table cells. removing this will require changes across several PR's that all used this `height` function. we may have to leave this for now.

Reply:
Yes, let's leave it for now but please add a comment here to notify us later that this is a temporary measure.